api/types/swarm/runtime.go

```diff
@@ -0,0 +1,11 @@
+package swarm
+
+// RuntimeType is the type of runtime used for the TaskSpec
+type RuntimeType string
+
+const (
+	// RuntimeContainer is the container based runtime
+	RuntimeContainer RuntimeType = "com.docker.runtime.container"
```

Comment:
I think right now the one we're using is simply called `container`

@ehazlett There's a `naming.Runtime()` helper to extract the name automatically:
https://github.com/docker/swarmkit/blob/master/api/naming/naming.go#L38